controllers/controllers.js

```diff
@@ -114,4 +115,10 @@ exports.removeComment = (req, res, next) => {
     });
 };
 
-
+exports.getAllUsers = (req, res, next) => {
+  fetchAllUsers()
+    .then((users) => {
+      res.status(200).json({ users });
```

Comment:
Using `.json` after using `.send` so far - for consistency change back to `.send`

Reply:
done